lang/lang_en.txt

```diff
@@ -268,9 +244,6 @@
 #MSG_FSENSOR_ON c=0 r=0
 "Fil. sensor  [on]"
 
-#MSG_RESPONSE_POOR c=20 r=2
```

Comment:
Does that means that the PAT9125 and so the MK3 aren't supported anymore? or is that just the translation?